backend/src/routes/reset.ts

```diff
@@ -0,0 +1,55 @@
+import express from 'express';
+const mysql = require('mysql2/promise');
+
+const router = express.Router();
+
+// AWS RDS Configuration
+let db_connection = mysql.Connection;
+(async () => {
+  try {
+      db_connection = await mysql.createConnection({
+      host: process.env.AWS_RDS_ENDPOINT,
+      user: process.env.AWS_RDS_USERNAME, 
+      password: process.env.AWS_RDS_PASSWORD, 
+      database: process.env.AWS_RDS_DATABASE_NAME,
+      port: parseInt(process.env.AWS_RDS_PORT as string, 10)
+    });
+    // console.log('Database connection established successfully');
+  } catch (error) {
+    // Log the environment variables to ensure they are being read correctly
+    console.log('Database configuration:', {
```

Comment:
Not sure if we should log some secure passwords to the console.

Reply:
@bitNathan

Reply:
updated, was using it for testing since the SQL database was being difficult last week